Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -727,6 +726,6 @@ public void RaiseDictationError(IInputSource source, uint sourceId, string dicta
         }
 
         #endregion // Dictation Events
-
+#endif
```

Comment:
Sorry took a bit longer had to install unity on my machine at home. I tested it earlier on my workstation. I get the errors with android as the target platform.